message/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from player.models import Player
+from game_event.models import GameEvent
+
+# Create your models here.
+
+
+class Message(models.Model):
+    user_id = models.ForeignKey(Player, on_delete=models.CASCADE)
+    game_event_id = models.ForeignKey(GameEvent, on_delete=models.CASCADE)
+    time_sent = models.DateTimeField(auto_now_add=True)
+    message_text = models.CharField(max_length=255)
```

Comment:
Do we really need to name it `message_text `? Since it is already a field in the Message class, it seems obvious that we are referring to the message itself. what do you think?

Reply:
@Yuval-Vino You're right here it is stated that it is a 'charField', but in other places in our code, we are accessing this attribute, if we omit the "_text" part of this attribute name, it could be a bit confusing just to try access message attribute in a  message entity. Wdyt ?

Reply:
I was actually thinking of removing the message part from the field and just calling it text so you can refer to it as Message.text

Reply:
@Yuval-Vino Seems like a good idea, working on it.